docs/sources/observability-as-code/provision-resources/git-sync-setup.md

```diff
@@ -143,19 +153,8 @@ In this step you can decide which elements to synchronize. Keep in mind the avai
 - Choose **Sync external storage to new Grafana folder** to sync external resources into a new folder without affecting the rest of your instance. You can repeat this process for up to 10 connections.
 
 1. Enter a **Display name** for the repository connection. Resources stored in this connection appear under the chosen display name in the Grafana UI.
-1. Click **Synchronize** to continue.
-
-<!-- ### Synchronize with external storage
-
-{{< admonition type="note">}}
```

Comment:
This you removed is important for the instance sync, when there are pre-existing resources in the instance. History is also an option of instance sync, not support by the folder one. We should probably keep this, but find it a new home. Likely the Instance Sync limitations.